ui/raidboss/data/05-shb/raid/e8n.js

```diff
@@ -6,8 +6,520 @@
+      },
+    },
+    {
+      id: 'E8N Absolute Zero',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DD7', capture: false }),
+      regexDe: Regexes.startsUsing({ source: 'Shiva', id: '4DD7', capture: false }),
+      regexFr: Regexes.startsUsing({ source: 'Shiva', id: '4DD7', capture: false }),
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DD7', capture: false }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DD7', capture: false }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DD7', capture: false }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E8N Double Slap',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DDA' }),
+      regexDe: Regexes.startsUsing({ source: 'Shiva', id: '4DDA' }),
+      regexFr: Regexes.startsUsing({ source: 'Shiva', id: '4DDA' }),
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DDA' }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DDA' }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DDA' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'E8N Frigid Water',
+      regex: Regexes.headMarker({ id: '0057' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Flare on YOU',
+        de: 'Flare auf DIR',
+      },
+    },
+    {
+      id: 'E8N Icicle Impact',
+      regex: Regexes.headMarker({ id: '0060' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'E8N Puddle Chase',
+      regex: Regexes.headMarker({ id: '00C5' }),
+      condition: Conditions.targetIsYou(),
+      alertText: {
+        en: 'Dropping puddles--Stay out',
+      },
+    },
+    {
+
+      id: 'E8N Stoneskin',
+      regex: Regexes.startsUsing({ source: 'Earthen Aether', id: '4DEF' }),
+      regexDe: Regexes.startsUsing({ source: 'Erdäther', id: '4DEF' }),
+      regexFr: Regexes.startsUsing({ source: 'éther de terre', id: '4DEF' }),
+      regexJa: Regexes.startsUsing({ source: 'アース・エーテル', id: '4DEF' }),
+      // FIXME: CN/KO translations.
+      condition: function(data) {
+        return data.canSilence();
+      },
+      // It's a minor annoyance at best, so don't bug the user too much.
+      response: Responses.interrupt('info'),
+    },
+    {
+      id: 'E8N Holy',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DEC', capture: false }),
+      regexDe: Regexes.startsUsing({ source: 'Shiva', id: '4DEC', capture: false }),
+      regexFr: Regexes.startsUsing({ source: 'Shiva', id: '4DEC', capture: false }),
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DEC', capture: false }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DEC', capture: false }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DEC', capture: false }),
+      response: Responses.getOut(),
```

Comment:
Can you differentiate the getOut from the getIn with a different alert level? maybe info/alert (or alert/alarm?) for out/in since I've seen many people get faked out by this.

Reply:
That's fair, I suppose. Info OUT/alert IN, I think, since OUT is the default.

(This set of triggers is also a bit uncomfortable for me personally, as it's a clear example of ACT letting players anticipate a choice they wouldn't otherwise be able to unassisted.)